CONTRIBUTING.md

```diff
@@ -64,3 +64,55 @@ Other Tips
     your existing commit message by running:
 
         $ git commit --amend
+
+Checklist for Contributed Code
+------------------------------
+
+A pull request for a new feature will be reviewed to see if it meets the following requirements.  For any pull request, an astropy maintainer can help to make sure that the pull request meets the requirements for inclusion in the package.  
+
+**Scientific Quality**
+  * Is the submission relevant to astronomy? 
+  * Are references included to the origin source for the algorithm?
+  * Does the code perform as expected?
+  * Has the code been tested against previously existing implimentations?
+
+**Code Quality**
+  * Are the [coding guidelines](http://docs.astropy.org/en/latest/development/codeguide.html)
+    followed?
+  * Is the code compatible with Python 2.6, 2.7, as well as 3?
+  * Are there dependancies other than the Astropy core, the Python Standard 
+    Library, and NumPy 1.6.0 or later?
+    * Is the package importable even if the C-extensions are not built?
+    * Are additional dependancies handled appropriately?
+    * Do functions that require additional dependancies  raise an `ImportError`
+        if they are not present?
+  
+**Testing**
+  * Are the [testing guidelines](http://docs.astropy.org/en/latest/development/testguide.html) followed?
+  * Are the inputs to the functions sufficiently tested?
+  * Are their tests for any exceptions raised?
+  * Are there tests for the expected performance?
+  * Are the sources for the tests documented?
+  * Have tests that require an [optional dependancy marked](http://docs.astropy.org/en/latest/development/testguide.html#tests-requiring-optional-dependencies) as such?
+  * Does python setup.py test run without failures?
+
+**Documentation**
+  * Are the [documentation guidelines](http://docs.astropy.org/en/latest/development/docguide.html) followed? 
+  * Is there a [docstring](http://docs.astropy.org/en/latest/development/docrules.html) in the function describing:
+    * What the code does?
+    * The format of the inputs to the function?
+    * The format of the outpus of the function?
+    * References to the original algorithms?
+    * Any exceptions which are raised?
+    * An example of running the code?
+  * Is there any information needed to be added to the docs to describe the function?
+  * Does the documentation build without errors or warnings?
+
+**License**
+  * Is the astropy license included at the top of the file?
+  * Are there any conflicts with this code and existing codes? 
+
+**astropy requirements**
+  * Do all the tests pass on the travis build?
+  * Has an entry been added into the changelog?
```

Comment:
*if applicable*.  Maybe I can clarify when that is applicable (I know I put something about that recently in CONTRIBUTING.md)